pkg/cvo/cvo_scenarios_test.go

```diff
@@ -2864,19 +2906,6 @@ func verifyAllStatus(t *testing.T, ch <-chan SyncWorkerStatus, items ...SyncWork
 	}
 }
 
-func waitFor(t *testing.T, fn func() bool) {
```

Comment:
+1 to dropping this; looks like it was added in 961873d66a (#82) with no consumers then or since.